ostia-operator/pkg/apicast/apicast.go

```diff
@@ -208,3 +210,31 @@ func asOwner(api *v1alpha1.API) metav1.OwnerReference {
 func apicastName(api *v1alpha1.API) string {
 	return "apicast-" + api.Name
 }
+
+func newLivenessProbe() *v1.Probe {
```

Comment:
Can you merge those two functions into "newProbe()" ? and pass the required parameters, either the Handler or just Path, Port, and the timeouts...